R/scan_data.R

```diff
@@ -32,56 +32,65 @@
 #'data_20 = scan_data(stations, -20)
 #'
 #'#40 inch
-#'data_20 = scan_data(stations, -40)
+#'data_40 = scan_data(stations, -40)
 #'}
 #'@export
 scan_data = function(stationTriplets, depth, start=Sys.Date()-as.difftime(1, units='days'), end=Sys.Date()){
 	
+  returns = data.frame()
+  stationChunks = split(stationTriplets, ceiling(seq_along(stationTriplets)/5))
```

Comment:
The only caveat is that it should be split in a way that slivers of data are missed, but I don't know enough about the guts of the api to know how big of a risk that is.